server/container_create.go

```diff
@@ -766,9 +766,6 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	specgen.SetRootPath(mountPoint)
 
 	saveOptions := generate.ExportOptions{}
-	if err = specgen.SaveToFile(filepath.Join(containerInfo.Dir, "config.json"), saveOptions); err != nil {
```

Comment:
@mrunalp I think the correct PR would drop `RunDir` config.json. We'll always read from `Dir` instead which works with restore after reboot. We need to not read from RunDir anymore though, @nalind is this something feasible? Does container create work regardless?